### PR TITLE
Updated eventuate rabbitmq messaging version and used new address configuration.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,5 +8,6 @@ springBootVersion=2.1.1.RELEASE
 
 version=0.1.0-SNAPSHOT
 eventuateCommonImageVersion=0.10.0.RELEASE
-eventuateCdcImageVersion=0.7.0.RELEASE
-eventuateTramVersion=0.25.1.RELEASE
+eventuateMessagingKafkaImageVersion=0.9.0.RELEASE
+eventuateCdcImageVersion=local-test
```
Comment (cer) [inline gradle.properties]: What's this?

File: gradle.properties [hunk 2]
```diff
@@ -8,5 +8,6 @@ springBootVersion=2.1.1.RELEASE
...
+eventuateTramVersion=0.27.0-SNAPSHOT
```
Comment (cer) [inline gradle.properties]: And this?

Comment (dartartem) [reply gradle.properties]: Forgot to add BUILD to it

File: gradle.properties (hunk 1 above)
Comment (dartartem) [reply gradle.properties]: we don't have BUILD_SNAPSHOT versions for images, right? So I used local build, it should be changed when new version is released.

**Pull request merged**